src/libraries/System.Linq/tests/ToArrayTests.cs

```diff
@@ -331,6 +331,9 @@ public void NonConstantTimeCountEmptyPartitionSelectDiffTypeToArray()
         }
 
         [Theory]
+#if OUTERLOOP
```

Comment:
It should be fine move it to the outer loop everywhere. No need to special case it.